### ISSUE TITLE
[Slider, ColorSlider, ColorArea, ColorWheel, ColorHandle] "touch-action: none" missing in a number of patterns

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-css#1189]
## Description
`Slider`, `ColorSlider`, `ColorArea`, `ColorWheel`, `ColorHandle` all deserve to have their `touch-action` property manages so that trying to set the value of those interfaces doesn't cause page scrolling in touch context.

## Expected behavior
Setting values, and touching the "track", on these elements do not scroll the page in touch environments.

refs  https://github.com/adobe/spectrum-web-components/issues/1454

**Comment by castastrophe** [comment]
Reference: https://github.com/adobe/spectrum-web-components/pull/1455/files

**Comment by castastrophe** [comment]
Closing this since we have those styles covered now in SWC.
